src/OmniSharp.Roslyn/OmniSharp.Roslyn.csproj

```diff
@@ -16,6 +16,7 @@
     <PackageReference Include="Microsoft.Extensions.Caching.Memory" />
     <PackageReference Include="Microsoft.CodeAnalysis.Common" />
     <PackageReference Include="System.ComponentModel.Composition" />
+    <PackageReference Include="System.Reflection.DispatchProxy" Version="4.5.1" />
```

Comment:
please add the reference here https://github.com/OmniSharp/omnisharp-roslyn/blob/master/build/Packages.props and then use it without a version in the csproj